qng/common.py

```diff
@@ -0,0 +1,69 @@
+import argparse
+import json
+import operator
+import os
+import random
+import sys
+import unicodedata
+import re
+from typing import Optional
+
+import qng
+
+
+_BASE_DIR = os.path.dirname(os.path.realpath(__file__))
+_DATA_DIR = os.path.join(_BASE_DIR, 'data')
+
+
+def _validate_snake_kebab_args(args):
```

Comment:
Not sure where else to put this in the file, but personally to denote a module that's meant to be used across the package but isn't part of the public API, I'd call the file `_common.py` and lose the underscore prefix on the function names, _unless_ they're only used in the _common module itself.

Just a stylistic preference though, but that way I can distinguish what's internal to `_common` vs. what's internal to the package but accessible outside `_common`.